data/json/monsters.json

```diff
@@ -5729,6 +5765,7 @@
     "armor_bash":0,
     "armor_cut":0,
     "vision_day":30,
+    "vision_night":10,
```

Comment:
I've seen enough Thriller reports on the forum to know that 1:10K odds still happen; granted, Thriller is in town and ACSL is not, so more folks have opportunities to encounter Thriller.

Nevertheless, I'd rather not have to read folks on the forum complaining that a red @ came out of nowhere at night and stabbed 'em to death as they were walking through the woods.  Tank drones, for all the complaints, were more civilized than that.